src/1-raw-data/gcs/gcs.go

```diff
@@ -80,19 +89,23 @@ func (gcs *StorageConnection) CreateGCSBucket() error {
 }
 
 // Store files in storage.
-func (gcs *StorageConnection) StoreGCS(fileName, url string) error {
+func (gcs *StorageConnection) StoreGCS(ctx context.Context, fileName, url string) error {
```

Comment:
This method seems to store the response body from the given URL in storage? Is that accurate? 

Consider updating the comment and/or renaming the function to make that a little clearer `gcs.DownloadToBucket` or something

Reply:
Good points